### PR TITLE
Create Blog “ilorest-as-a-pypi-package”

### PR DESCRIPTION
**Pull request by rajeevkallur**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by choukden — CHANGES_REQUESTED** [review]
Hi Rajeev,
Thank you for your new contribution.
Please find here a few comments and suggestions for changes in order for the Netlify process to build the preview of your blog post.

Thanks
Denis

File: content/blog/ilorest-as-a-pypi-package.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+   $ find / -name ilorest
+   ```
+   
+NOTES:
+
+- Pypi package can also be used for ARM-based operating systems if Python 3 is present.
+
+- Additionally, the ILORest pypi package can be utilized on RHEL, SLES, and Windows platforms as long as Python 3 is installed (preferably version > 3.8).
+
+- In a air-gapped environment, the pypi package can be downloaded from pypi.org and installed using the following command. Dependencies will need to be installed separately.
```
Comment (donzef) [inline content/blog/ilorest-as-a-pypi-package.md:62]: I would rephrase:

\- in an air-gapped environment, the HPE iLOrest PyPI package can be downloaded from the \<a href="https://pypi.org/project/ilorest/" target="_blank">PyPI repository\</a> and installed using the following command. Dependencies may nee to be installed separately.

File: content/blog/ilorest-as-a-pypi-package.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+   ```shell
+   $ pip3 install <path to the downloaded pypi package>
```
Comment (donzef) [inline content/blog/ilorest-as-a-pypi-package.md:65]: Change "pypi" into "PyPI".

File: content/blog/ilorest-as-a-pypi-package.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+   ```
+
+
```
Comment (donzef) [inline content/blog/ilorest-as-a-pypi-package.md:68]: It would be good to add a paragraph to uninstall this PyPI package and its associated library, which, again, cannot co-exist with the DMTF Redfish library. Something like:

 To perform a clean removal of the HPE iLOrest PyPI package, don't forget to uninstall the associated Python library as mentioned in the following command:

\```shell
$ pip uninstall --yes python-ilorest-library ilorest 
\```

**Review by donzef — CHANGES_REQUESTED** [review]
Hi Rajeev,

Thanks to promote your "baby" via an HPE DEV blog post!

In order to be compliant with Marketing rules (HPE and others), it is highly recommended to stick to "HPE iLOrest", "PyPI", "macOS", "Microsoft Windows" or "MS Windows" and "Debian" names.

Then, I am suggesting to add external links to provide a better experience to the technical readers willing to get more detail on specific topics like the CHIF driver and the location of the PyPI packages.

Lastly, based on customer feedback,  I am proposing to explicitly mention the absence of co-existence of our Python iLOrest library and the DMTF's Redfish library.

File: content/blog/ilorest-as-a-pypi-package.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+---
+<style>
+li {
+   font-size: 27px;
+   line-height: 33px;
+   max-width: none;
+}
+</style>
+
+The HPE iLOrest tool has been repackaged into both source and binary distributions and is now available on [PyPI](https://pypi.org/project/ilorest/). This means it can be easily utilized on any operating system that has Python 3 installed. The intention is for the PyPI package to replace the existing builds for macOS, Debian and Ubuntu distributions of HPE iLOrest.
```
Comment (drensing) [inline content/blog/ilorest-as-a-pypi-package.md:23]: Hello, Rajeev!
This looks like a great blog post. My review is mostly from a grammar and Brand perspective, but I also help in terms of readability and the ability to draw readers in so you get as much traffic as possible.

While you technically have an introduction telling what the blog post is about, it could use a little more "oomf", meaning that you could provide a few more details around why this is an exciting thing - why should folks read this post?

You may get folks who already use iLOrest and feel that that's basically the audience you want to address. But what if there might be others out there who are considering using it but don't have as much background on it? Are there a few words you could add that might draw their attention to your post? I'm not suggesting "iLOrest is the coolest thing since sliced bread", but maybe something like "If you're looking for a great tool to do such-and-such, iLOrest gives you xyz. And for those of you who love working in Python, we have some great news for you..."

You are missing a summary/conclusion at the end. I'll get to that later ;-)

File: content/blog/ilorest-as-a-pypi-package.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+   ```shell
+   $ iLOrest -v login
+   ```
+7. You can check the location of iLOrest using the command:
+   ```shell 
+   $ find / -name iLOrest
+   ```
+   
+NOTES:
```
Comment (drensing) [inline content/blog/ilorest-as-a-pypi-package.md:57]: I see here that it says NOTES: but there's only numbered bullets after it. That makes me think that either some text is missing OR that some of these things that show up as bullets shouldn't actually be bullets.
image

File: content/blog/ilorest-as-a-pypi-package.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+* PyPI package can also be used for ARM-based operating systems if [Python 3](https://www.python.org/downloads/) is present.
+
+- Additionally, the HPE iLOrest PyPI package can be utilized on RHEL, SLES and Microsoft Windows platforms as long as [Python 3](https://www.python.org/downloads/) is installed (preferably version > 3.8).
+   
+
+* DMTF's [redfish](https://pypi.org/project/redfish/) library can not coexist with HPE [python ilorest library](https://pypi.org/project/python-ilorest-library/) which is a dependency for the HPE iLOrest PyPI package. So, make sure to remove the [redfish](https://pypi.org/project/redfish/) library using the command below:
```
Comment (drensing) [inline content/blog/ilorest-as-a-pypi-package.md:64]: Redfish and Python should have initial capital letters.
I'd recommend modifying the second sentence to:

So, make sure to remove any Redfish library you may have installed prior to installing the HPE Python iLOrest library using the command shown below:

File: content/blog/ilorest-as-a-pypi-package.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+   $ pip3 uninstall redfish
+   ```
+
+* In an air-gapped environment, the HPE iLOrest PyPI package can be downloaded from the <a href="https://pypi.org/project/ilorest/" target="_blank">PyPI repository</a> and installed using the following command. Dependencies may need to be installed separately.
+
+   ```shell
+   $ pip3 install <path to the downloaded PyPI package>  
+   ```
+  
+* To perform a clean removal of the HPE iLOrest PyPI package, don't forget to uninstall the associated Python library as mentioned in the following command:
```
Comment (drensing) [inline content/blog/ilorest-as-a-pypi-package.md:76]: I recommend you add a summary/conclusion along with a call to action. It could read something like this:

By following the instructions I've outlined above, you can start using the HPE iLOrest tool on PyPI making [your life so much better - or whatever it really gives you.] It's really quick and easy to get started. For more information on HPE iLO, along with some tips and tricks in working with it, make sure you check out the HPE Developer blog regularly. [and then link to https://developer.hpe.com/blog/  ].

**Review by drensing — CHANGES_REQUESTED** [review]
Hi there! I hope you find my comments/suggestions helpful. Thank you so much for your submission!

Dale

**Comment by donzef** [comment]
Not sure where we are in this publication process, but, I'll take my chance to chime in an propose my help to fulfill Dale's propositions to post  a more structured publication concerning :

* intro
* body structure
* Conclusion

@rajeevkallur , let me know.
